Feature/recycle instructions image

Raquelhill opened the pull request:
### Features
-N/A

### Refactors
-N/A
### Issues and Technical Debt
-N/A

### Other Comments, Concerns, or Questions
-N/A

### Checklist
- [X] If this is a feature branch, I have requested reviews from at least one FE team member and one BE team member
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature work
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran Lighthouse and WAVE accessibility audits and fixed errors/warnings
- [ ] Any dependent changes have been merged and published in downstream modules

Raquelhill committed:
Create .env folder

Raquelhill committed:
Import image the explains how to recycle

Raquelhill committed:
Rename varibales for developer readability

anthony-iacono commented on .env:
```diff
@@ -0,0 +1 @@
+REACT_APP_MAP_KEY=AIzaSyDq-vetDxJ2Jyq7CY8kbzN097Vpz4IF2nk
```
```suggestion
REACT_APP_MAP_KEY=AIzaSyDq-vetDxJ2Jyq7CY8kbzN097Vpz4IF2nk

```

anthony-iacono reviewed (changes requested):
@Raquelhill Will this component have any additional functionality beyond displaying an image? Will it be reused anywhere on the site? If not, what do you think about adding it to the LandingPage component instead?

ealwafai reviewed (commented):
Looks great, can't wait to see what it looks like

Raquelhill committed:
remove recycling image component, add image to the landing page

sarahrudy commented on src/placeDetails.js:
```diff
@@ -0,0 +1,258 @@
+        types: ['route'],
+      },
+      {
+        long_name: 'Pyrmont',
```
Great that we have this example response for now.

sarahrudy committed:
Update Map.js

Uncommented and removed import of placeDetails for this branch. We'll keep working on the map on a separate branch.

sarahrudy committed:
Delete placeDetails.js

Remove for now.

sarahrudy committed:
Update .env

Co-authored-by: Anthony Iacono <72999840+anthony-iacono@users.noreply.github.com>

sarahrudy committed:
Update Map.js

Raquelhill committed:
Fix merge conflicts

Raquelhill committed:
Finish fixing merge conflicts
